src/composables/Project.js

```diff
@@ -69,3 +87,68 @@ export function deleteProjectById(projectId) {
 export function deleteAllProjects() {
   localStorage.setItem(PROJECT_STORAGE_KEY, '{}');
 }
+
+/**
+ * Update remote origin, fetch and checkout the default branch.
+ * @param {Project} project - Project to update.
+ * @return {Promise<void>} Promise with nothing on success otherwise an error.
+ */
+export function updateGitProject(project) {
+  const dir = `/${project.id}`;
+
+  return git.addRemote({
```

Comment:
Use async/await instead of chaining then. The current syntax makes it difficult to understand what is effectively being returned by the function.

Reply:
I will repeat myself for this comment and all next comment. But, we want to use promise instead of async/await, because I prefere promise instead of async/await.
All library that I have to use, the good is one that return promise and the bad don't return promise.

Reply:
See with you, improve readability and remove promise chaining.
Done